config.c

```diff
@@ -445,6 +445,11 @@ static int pv_config_load_file(char *path, struct pantavisor_config *config)
 	config_iterate_items_prefix(&config_list, config_sysctl_apply,
 				    "sysctl.", NULL);
 
+	config->vm.oom_qos_besteffort = config_get_value_int(
+		&config_list, "vm.oom_qos_besteffort", 1000);
```

Comment:
I don't like the names very much, but I cannot think of something better either :)

Reply:
we can add a . in the middle, but then i dont know the benefit :)